### ISSUE TITLE
next/previous commenting range command doesn't work

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#193728]
I'm on a mac. setting it to be `option+f10` works though

**Comment by meganrogge** [comment]
I realized they only work if I've made a comment in the particular editor.

**Comment by meganrogge** [comment]
it started to work the moment I had at least one comment in the edtior

**Comment by alexr00** [comment]
@meganrogge did the original keybinding work after you added comment? Even before adding a comment I'm seeing the command work.

**Comment by meganrogge** [comment]
that keybinding still doesn't work for me (maybe my keyboard or something)...? but also, when I use a different keybinding, it does try to run the command. I'm now seeing this error.

```
Cannot read properties of undefined (reading 'getActiveRange')
``

![Screenshot 2023-09-22 at 8 18 37 AM](https://github.com/microsoft/vscode/assets/29464607/5e12c8b2-4c12-43c8-ba84-94e3c16a0e0f)

**Comment by alexr00** [comment]
I have confirmed that the default keyboard shortcut works on mac: `cmd+k cmd+option+down-arrow`.

There's probably something up with your keybindings that's causing this. You could try the "Developer: Toggle Keyboard Shortcuts Troubleshooting" command and see what it outputs.

I'm fixing this error though: https://github.com/microsoft/vscode/issues/193728#issuecomment-1731600635

**Comment by meganrogge** [comment]
removed bug because tracking in that other issue
